drivers/iscsilib.py

```diff
@@ -54,7 +53,7 @@ def doexec_locked(cmd):
     """Executes via util.doexec the command specified whilst holding lock"""
     _lock = None
     if os.path.basename(cmd[0]) == 'iscsiadm':
-        _lock = lock.Lock(LOCK_TYPE_RUNNING, 'iscsiadm')
+        _lock = lock.Lock(lock.LOCK_TYPE_GC_RUNNING, 'iscsiadm')
```

Comment:
This is now incorrect. IN this case the lock should be a unique `LOCK_TYPE_ISCSIADM_RUNNING` as it is a lock around usage of iscsiadm and nothing to do with the GC.